nanome_workspace_manager/WorkspaceManager.py

```diff
@@ -207,22 +201,78 @@ def update(self):
         if timer() - self.__timer >= 5.0:
             self.refresh_menu()
             self.__timer = timer()
+    
+    def load_accounts(self):
+        try:
+            with open(AUTH_PATH, 'r') as f:
+                self.accounts = [line.rstrip().split(' ') for line in f.readlines()]
```

Comment:
And what if there isn't 3 items in the line? We should either generate default values or display an error message